include/cape_ros/cape_ros_node.h

```diff
@@ -0,0 +1,88 @@
+/*
+ * This file is part of the cape_ros.
+ * Copyright (c) 2019 Avidbots Corp.
+ *
+ * This program is free software: you can redistribute it and/or modify
+ * it under the terms of the GNU General Public License as published by
+ * the Free Software Foundation, version 3.
+ *
+ * This program is distributed in the hope that it will be useful, but
+ * WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the GNU
+ * General Public License for more details.
+ *
+ * You should have received a copy of the GNU General Public License
+ * along with this program. If not, see <http://www.gnu.org/licenses/>.
+ */
+
+#ifndef CAPE_ROS_CAPE_ROS_NODE_H
+#define CAPE_ROS_CAPE_ROS_NODE_H
+
+#include <ros/ros.h>
+#include <sensor_msgs/CameraInfo.h>
+#include <sensor_msgs/Image.h>
+#include <cape_ros/Planes.h>
+#include <cv_bridge/cv_bridge.h>
+#include <cape_ros/cape.h>
+#include <opencv2/opencv.hpp>
+
+class CapeRosNode
+{
+public:
+  CapeRosNode(ros::NodeHandle nh);
+
+  void simpleProjectPointCloud(cv::Mat& X_, cv::Mat& Y_, cv::Mat& Z, Eigen::MatrixXf& cloud_array_,
```

Comment:
This does not match the function in cape_ros_node.cpp. Remove the trailing underscores of variable names. (X_, Y_, and cloud_array_)

Reply:
The style followed in cape_ros_node.cpp and cape_ros_node.h is the standard ros style http://wiki.ros.org/CppStyleGuide. Eventually the full package should be reformatted to that style.

Reply:
I'm referring to lines 75 & 76 of cape_ros_node.cpp:
`void CapeRosNode::simpleProjectPointCloud(cv::Mat& X, cv::Mat& Y, cv::Mat& Z, Eigen::MatrixXf& cloud_array, const double& depth_cutoff)`
Compared to this line:
`void simpleProjectPointCloud(cv::Mat& X_, cv::Mat& Y_, cv::Mat& Z, Eigen::MatrixXf& cloud_array_, const double& depth_cutoff);`

The arguments in the header are `X_`, `Y_`, `Z`, `cloud_array_`, and `depth_cutoff`, and the arguments in the source file are `X`, `Y`, `Z`, `cloud_array`, and `depth_cutoff`. The **trailing** underscores should be removed, since they are inconsistent between function definitions, plus it's weird to have three arguments with trailing underscores and two without in a function.
I understand using underscores is convention of ros style. I'm asking about the trailing underscore in a function argument.

Reply:
Oh ok, that's the result of automatic refactoring. will fix